.config/nextest.toml

```diff
@@ -13,7 +13,7 @@ dir = "target/nextest"
 # * retries = 3
 # * retries = { backoff = "fixed", count = 2, delay = "1s" }
 # * retries = { backoff = "exponential", count = 10, delay = "1s", jitter = true, max-delay = "10s" }
-retries = { backoff = "fixed", count = 2, delay = "1s", jitter = false }
+retries = { backoff = "fixed", count = 0, delay = "1s", jitter = false }
```

Comment:
This is a temporary change to make local test runs snappier. This will be removed from the final PR.